packages/manager/apps/telecom/src/app/telecom/pack/move/offers/move-offers.html

```diff
@@ -72,6 +74,22 @@
                                         data-ng-bind-html="offer.prices.price.price.text | tucFormatPrice"
                                     ></span>
                                 </div>
+
+                                <!-- Other promotion -->
+                                <div
+                                    data-ng-if="offer.prices.promotion && offer.prices.promotion.id === $ctrl.PROMO_DISPLAY.PROMO_FTTH_2020" class="alert alert-info mt-2">
+                                        <div
+                                            data-translate="pack_move_promotion_description"
+                                            data-translate-values="{ 'endDate' : (offer.prices.promotion.endDate | date : short) }"
+                                        >
+                                        </div>
+                                        <strong
+                                            data-ng-if="offer.prices.promotion.subscription.duration === $ctrl.PROMO_DISPLAY.P3M.key"
+                                            data-translate="pack_move_promotion_subscription_months_offered"
+                                            data-translate-values="{ 'duration' :  $ctrl.PROMO_DISPLAY.P3M.value }"
+                                        >
+                                        </strong>
```

Comment:
This looks really similar to the previous one, maybe having a component could help avoiding duplication and ease maintenance

Reply:
yes it's a good idea. I note this comment for a next rework on this part.